frame/balances/src/lib.rs

```diff
@@ -2189,3 +2189,11 @@ where
 		Self::update_locks(who, &locks[..]);
 	}
 }
+
+pub struct BalancesTotalIssuance<T, I = ()>(sp_std::marker::PhantomData<(T, I)>);
```

Comment:
`TotalIssuance` name is taken by the storage and can't think a better name. Maybe we should simply implement `Get` for `StorageValue`?

Reply:
>  Maybe we should simply implement Get for StorageValue?

`Get` is already ambiguous regarding being constant or not, I think what you did now is better. 

We are thinking of deprecating `Get` types as well.